packages/react/react/src/hooks/useResizeObserver.ts

```diff
@@ -21,7 +21,7 @@ export type OnResize = (entry: ResizeObserverEntry) => void;
  *
  * <button ref={ref}>...</button>
  */
-export function useResizeObserver<E extends HTMLElement = HTMLElement>(onResize: OnResize) {
+export default function useResizeObserver<E extends HTMLElement = HTMLElement>(onResize: OnResize) {
```

Comment:
This is not necessary change in this PR.

Reply:
Thanks for your suggestion.

I already have stashed files that fix all hooks to be exported in the same way(default export).